package.json

```diff
@@ -11,22 +11,27 @@
     "lint": "eslint src/*.js"
   },
   "devDependencies": {
+    "autoprefixer": "^9.6.1",
     "css-loader": "^2.1.1",
     "file-loader": "^4.0.0",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
-    "path": "^0.12.7",
     "mini-css-extract-plugin": "^0.6.0",
     "node-sass": "^4.12.0",
+    "path": "^0.12.7",
```

Comment:
I feel you don't need `path` since it's a built-in package.

Reply:
https://github.com/trust2065/orchard/blob/74125c2e121a0fd1f7c96b1e76c3293b5a26a909/webpack.config.js
I need it in line 9 and line 62.

Reply:
I know. Just don't feel that you need to npm install a built-in package. Webpack is nodejs.
 https://www.w3schools.com/nodejs/ref_modules.asp

Reply:
Cool, I am going to remove it.